src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -912,7 +938,38 @@ private void updateEditState() {
 
             mEditState.setEdits(editMap);
             mSeenExperiments.addAll(toTrack);
-            mVariantTracker.trackVariants(toTrack);
+
+            if (toTrack.size() > 0) {
```

Comment:
It was an opportunity to simplify a bit - VariantTracker was never used other than here. Originally, I wanted to eliminate the mutual dependency with MixpanelAPI, but it turned out not to be practical and VariantTracker was never removed.